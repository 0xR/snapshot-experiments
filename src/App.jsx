import React from 'react';

import styles from './App.css';
import logo from './logo.png';

export default () => (
  <div>
    <img src={logo} alt="logo" />
    <h1 className={styles.heading}>Hello from app</h1>
    <p className={styles.paragraph}>This is a paragraph</p>
    <input />
  </div>
);
