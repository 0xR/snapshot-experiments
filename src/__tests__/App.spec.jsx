import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App.jsx';

describe('App component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('works with chai', () => {
    expect({ a: 1 }).to.deep.equal({ a: 1 });
    expect({ a: 1 }).not.to.deep.equal({ a: 2 });
  });
});
