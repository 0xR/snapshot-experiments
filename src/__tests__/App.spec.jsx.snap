exports[`App renders correctly`] = `
<div>
  <img
    alt="logo"
    src="logo.1731a1ce.png" />
  <h1
    className="App__heading___l4-3T typography__default-font___1ZFpu colors__font-color___12YaD">
    Hello from app
  </h1>
  <p
    className="App__paragraph___-3KUq typography__default-font___1ZFpu colors__font-color___12YaD">
    This is a paragraph
  </p>
  <input />
</div>
`;
